low-level/framework/base/sspl_constants.py

```diff
@@ -156,16 +156,23 @@
             "MESSAGE_HANDLERS": [],
             "SENSORS": ["CPUFaultSensor"],
         },
+        "SERVICEMONITOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": [],
```

Comment:
Add sensor name in list. "SENSORS": ["ServiceMonitor"],

Reply:
Then probably I will have to remove it from "_ENABLE_ALWAYS : SENSORS/DEGRADED_STATE_MODULES"..

Reply:
>> I didn't add since I saw the same trend in SYSTEMDWATCHDOG entry.
Okay, then you can keep as it is.

Reply:
👍